internal/monitors/elasticsearch/metadata.yaml

```diff
@@ -0,0 +1,182 @@
+- dimensions: null
+  doc: |
+    Monitors ElasticSearch instances.
+  metrics:
+  - description: The total number of get requests since node startup
```

Comment:
Go ahead and add a `group: ` field to all of these and extend the schema in https://github.com/signalfx/signalfx-agent/blob/master/internal/selfdescribe/monitors.go#L20 to support it.  It will save @jaysfx some work later and will be good to get while everything if fresh on your mind.

Reply:
Also, is this list really exhaustive?  It seems too short given how much code there was.

Reply:
This is not exhaustive. I made use of the file from the existing monitor. Looks like this is only a fraction of all the metrics. Will update